internal/cluster/validator.go

```diff
@@ -473,38 +355,22 @@ func isReadyToInstall(status string) bool {
 	return funk.ContainsString(allowedStatuses, status)
 }
 
-func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext) ValidationStatus {
+func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext) (ValidationStatus, string) {
 	readyToInstall := true
 	for _, host := range c.cluster.Hosts {
 		readyToInstall = readyToInstall && isReadyToInstall(swag.StringValue(host.Status))
 	}
```

Comment:
```suggestion
	readyToInstall := true
	for _, host := range c.cluster.Hosts {
		if !isReadyToInstall(swag.StringValue(host.Status)) {
		        readyToInstall = false
		        break
		}
	}
```

I know this is unchanged, but it seems strange to loop through all of the hosts if you already know one isn't ready.